run_tests.sh

```diff
@@ -28,4 +28,24 @@ else
     exit 1
 fi
 
-set +e
\ No newline at end of file
+set +e
+
+#Running the Docker infra test
+
+dock_ver="$(docker -v)"
+required_dock_ver="17.08.0"
+if [[ "${dock_ver[0]}" =~ "$required_dock_ver" ]]; then 
```

Comment:
Better to make this >=
rather than equal.
Atleast for now..
Offcourse when we know some latest version of docker fails for any reason, we can probably either just choose to upper bound the docker version(least likely) or modify our dockerfiles to suit the latest docker versions and accordingly raise the minimum supported docker version